components/resource_control/src/metrics.rs

```diff
@@ -4,22 +4,25 @@ use lazy_static::*;
 use prometheus::*;
 
 lazy_static! {
+    // TODO: deprecate the "name" label in v8.0.
```

Comment:
why not replace it directly, grafana panel should be updated with the version

Reply:
There is a problem that if we replace it, after a cluster is upgraded from old version, old metrics data will lost due to new expression. @nolouch  do you think we should keep this compatibility?

Reply:
IMO, the background relatives can be replaced directly due to no cases using it currently. if we do we need to pick to release-7.5